src/components/NavigationBrick/index.js

```diff
@@ -1,56 +1,103 @@
-import React from 'react';
+import React, { Component } from 'react';
 import styles from './styles';
 import { CallToAction } from '../../';
+import Fontawesome from 'react-fontawesome';
 
-function handleClick(onClick, args) {
-  return (event) => {
-    event.preventDefault();
-    onClick(args);
-  }
-}
+export default class NavigationBrick extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+			open : false
+		}
+	}
 
-function renderItems(s, items) {
-	if (items && items.length > 0) {
-		return items.map((item, index) => {
-			return (
-				<li style={ item.highlight ? s.itemHighlight : s.item } key={ index }>
-					<a style={ s.itemLink } href="#" onClick={ handleClick(item.onClick) }>{ item.label }</a>
-				</li>
-			);
-		});
-	} else {
-		return null;
+	handleClick(onClick, args) {
```

Comment:
you don't need this method, just call the onClick handler when you render the <button/>